help_to_heat/frontdoor/epc_api.py

```diff
@@ -27,4 +27,6 @@ def __api_call(self, url):
         headers = self._basic_auth_header()
         response = requests.get(url, headers=headers)
         response.raise_for_status()
+        if len(response.content) == 0:
+            return None
```

Comment:
the new API will return no content if nothing is found, this allows callers to handle it (otherwise the below .json() would fail)